### ISSUE TITLE
Setting `refreshEnabled` on a RefreshComposite disables all children

### ISSUE THREAD

**Issue by drew-wallace** [eclipsesource/tabris-js#1986]
### Problem description
I want to disable the pull-to-refresh gesture and only show it programatically. In the docs, it says setting `refreshEnabled` to false disables the gesture. Unfortunately, it also sets `enabled` to false on all children.

### Expected behavior
`refreshEnabled` should only control the gesture behavior and `enabled` should control any other interactions.

### Environment

- Tabris.js version: 3.2.0 
- Device: Google Pixel 3 
- OS: Android 10 

### Code snippet

```javascript
import {RefreshComposite, TextView, contentView, Button} from 'tabris';

contentView.append(
  <RefreshComposite stretch refreshEnabled={false} onRefresh={handleRefresh}>
    <TextView stretchX top={32} alignment='centerX' font='black 24px'>
      Pull to refresh
    </TextView>
    <TextView id='log' stretchX top='prev() 32' alignment='centerX' lineSpacing={1.4}/>
    <Button text="test" top="prev()" onSelect={() => $(RefreshComposite).only().refreshIndicator = true} enabled={true}/>
  </RefreshComposite>
);

const refreshComposite = $(RefreshComposite).only();
const log = $(TextView).only('#log');

function handleRefresh() {
  setTimeout(() => {
    refreshComposite.refreshIndicator = false;
    log.text = `last refresh: ${new Date()}\n${log.text}`;
  }, 1000);
}
```

**Comment by mpost** [comment]
Thank you for the bug report. You are right about your observation. `refreshEnabled` should only disable the gesture. We will fix the issue for the upcoming 3.3. release.

**Comment by mpost** [comment]
Issue has been resolved as advertised.
